### DISCUSSION TITLE
Is there a way to import invoices, settings and logo from Manta ?

### DISCUSSION THREAD

**Discussion by sonalder-darlene** [AndresMorelos/InvoncifyReleases#23] [Q&A]
**Desktop (please complete the following information if needed):**
 - OS: macOS
 - Version Last

**Question**
I have some invoices in Manta and all my settings + logo and informations that I would like to import to Invoicify, I've try going into the Application Support folder and copy pasting the files from Manta to Invoicify but it didn't do anything. Could you help ?

Nice to have a Import/Export in Invoicify I'm sure this feature will be extremely helpful. Thanks for maintaining this project !

**Comment by AndresMorelos** [comment]
@sonalder-darlene from now I can confirm that we need a  manually merge of the setting file located at `/Library/Application Support/Manta` with the settings located at `~/Library/Application Support/Invoncify`

There are some properties like `encryption` that will be needed by the Invoncify app that Manta doesn't have.

I keep looking a way to import the invoices from Manta and implement an automated way to import the Manta Settings to Invoncify.

**Reply by sonalder-darlene** [reply]
How can I do that ?
I've copy paste all the file from Manta to Invoncify and nothing...

That would be a nice feature to have importing from the previous app
